src/highdicom/sr/templates.py

```diff
@@ -385,11 +442,61 @@ def _contains_uidref_items(
     return False
 
 
+def _contains_image_items(
+    parent_item: ContentItem,
+    name: Union[Code, CodedConcept],
+    referenced_sop_class_uid: Union[str, None] = None,
+    referenced_sop_instance_uid: Union[str, None] = None,
+    relationship_type: Optional[RelationshipTypeValues] = None
+) -> bool:
+    """Checks whether an item contains a specific item with value type IMAGE.
```

Comment:
I've been trying to switch to using this form as specified by [PEP 0257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

```suggestion
    """Check whether an item contains a specific item with value type IMAGE.
```